wikimedia/wikipedia-ios
# Update Wikipedia logo alignment when user is logged out

**Phabricator:** https://phabricator.wikimedia.org/T304452

### Notes
When the user is logged in, use the existing centered title alignment for the navigation bar to display the notifications center bell and centered Wikipedia logo. But when the user is logged out, revert to the previous behavior of left/right aligning the Wikipedia logo via the large title display type.

### Test Steps
1. Log out, confirm the Wikipedia logo is edge aligned in the Explore feed.
2. Log in, confirm the Wikipedia logo is centered between the Notifications Center button and the Settings button in the Explore feed.

![logo_alignment](https://user-images.githubusercontent.com/5652327/162503814-b4d02f5c-ebfe-4291-ba0d-721e1a3bce46.jpg)
